test/integrations/destinations/ga4_v2/processor/customMappings.ts

```diff
@@ -540,6 +540,7 @@ export const customMappingTestCases = [
               },
               body: {
                 JSON: {
```

Comment:
can we have a test case for getting `sessionId` & `sessionNumber` also to be included ?

Reply:
Added